docs/DOCKER.rst

```diff
@@ -47,19 +39,28 @@ For running an S3 AWS backend, you will have to add a new section
             "credentialsProfile": "default"
         }
     }
+(...)
 
-You will also have to mount your AWS credentials file:
+You will also have to edit your AWS credentials file to be able to use your
+command line tool of choice. This file should only state credentials for S3 AWS
```

Comment:
this does not have to do with command line tools.  We use the aws credentials file to pull the credentials for the aws sdk client within cloud server.

Reply:
Yes, that's when we mount it. I meant that they need to input their credentials in there like they would for using S3. I've tried a new wording, let me know what you think.